cfme/tests/automate/test_service_dialog.py

```diff
@@ -481,3 +484,128 @@ def test_automate_methods_from_dynamic_dialog_should_run_as_per_designed(
+               Message: create, Request: object_walker)
+            4. Click the custom button, and observe the dialog. The element 'Text Box 1' default
+               value is empty, the dynamic element 'Text Box 2' has been dynamically populated.
+               click on submit button.
+            5. Repeat step 4, but type some value(like "aa") in element 'Text Box 1' and click on
+               Submit button.
+            6. Repeat step 5, but now amend the text dynamically entered into 'Text Box 2'
+               (for example adding the string "also").
+        expectedResults:
+            1.
+            2.
+            3.
+            4. Values passed through to object_walker - both element values should be passed:
+               >> ~/object_walker_reader.rb | grep dialog
+                     |    $evm.root['dialog_text_box_1'] =    (type: String)
+                     |    $evm.root['dialog_text_box_2'] = The dynamic method ran at:
+                     2019-05-30 09:42:40 +0100   (type: String)
+            5. The value from both test boxes should be passed through object_walker:
+                ~/object_walker_reader.rb -t 2019-05-30T09:43:25.558037 | grep dialog
+                     |    $evm.root['dialog_text_box_1'] = aa   (type: String)
+                     |    $evm.root['dialog_text_box_2'] = The dynamic method ran at:
+                     2019-05-30 09:42:43 +0100   (type: String)
+            6. Note that both element values are now passed through to object_walker:
+                ~/object_walker_reader.rb | grep dialog
+                     |    $evm.root['dialog_text_box_1'] = ccdd   (type: String)
+                     |    $evm.root['dialog_text_box_2'] = The dynamic method also ran at:
+                     2019-05-30 09:50:10 +0100   (type: String)Provision more than 10 VMs
+    """
+    sd, ele_label = import_dialog
+    button_grp, button = custom_button(dialog=sd)
+    request.addfinalizer(button_grp.delete_if_exists)
+    request.addfinalizer(button.delete_if_exists)
+
+    user_obj = appliance.collections.users.instantiate(name="Administrator")
+    view = navigate_to(user_obj, "Details")
+    custom_button_group = Dropdown(view, button_grp.hover)
+    view = navigation_to_view(custom_button_group, button)
+
+    # Step 4
+    with LogValidator("/var/www/miq/vmdb/log/automation.log",
+                      matched_patterns=[".*dialog_text_box_1:      .*",
+                                        f".*dialog_text_box_2: {view.text_box2.read()}.*"]
+                      ).waiting(timeout=120):
+        view.submit.click()
+
+    # Step 5
+    view = navigation_to_view(custom_button_group, button)
+    with LogValidator("/var/www/miq/vmdb/log/automation.log",
+                      matched_patterns=[".*dialog_text_box_1: Hello.*",
+                                        f".*dialog_text_box_2: {view.text_box2.read()}.*"]
+                      ).waiting(timeout=120):
+        view.text_box1.fill("Hello")
```

Comment:
optional: I would use random text with fauxfactory and try to validate as per it in log.